src/H5WebViewer.ts

```diff
@@ -32,7 +32,19 @@ export default class H5WebViewer
     webviewPanel: WebviewPanel
   ): Promise<void> {
     const { webview } = webviewPanel;
-    webview.options = { enableScripts: true };
+
+    // Allow opening files outside of workspace
+    // https://github.com/ucodkr/vscode-tiff/blob/master/src/tiffPreview.ts#L27-L30
```

Comment:
```suggestion
    // https://github.com/ucodkr/vscode-tiff/blob/9a4f976584fcba24e9f25680fcdb47fc8f97493f/src/tiffPreview.ts#L27-L30
```

I had too often such links to `master` not resolving to the relevant part cause `master` changed.